nrf_modem/doc/sockets.rst

```diff
@@ -275,17 +275,19 @@ NRF_SO_SEC_HANDSHAKE_STATUS
 
 NRF_SO_SEC_DTLS_CID
    Set DTLS Connection ID setting.
+   This socket option decides whether the modem will request or accept a DTLS connection ID when performing the server handshake.
    The socket option is supported from modem firmware v1.3.x, where x is greater than or equal to 5, and v2.x.x.
    The following values are accepted:
 
-   * :c:macro:`NRF_SO_SEC_DTLS_CID_DISABLED` - The DTLS connection ID is not used, and the connection ID extension is not included in the client hello.
-   * :c:macro:`NRF_SO_SEC_DTLS_CID_SUPPORTED` - The DTLS connection ID is supported but not used, and the connection ID extension with a zero-length CID is included in the client hello.
-   * :c:macro:`NRF_SO_SEC_DTLS_CID_ENABLED` - The DTLS connection ID is used, and the connection ID extension with a valid CID is included in the client hello.
+   * :c:macro:`NRF_SO_SEC_DTLS_CID_DISABLED` - The connection ID extension is not included in the client hello. Hence, the DTLS connection ID is not used.
+   * :c:macro:`NRF_SO_SEC_DTLS_CID_SUPPORTED` - The connection ID extension with a zero-length CID is included in the client hello. Hence, the modem will accept a DTLS connection ID from the server.
+   * :c:macro:`NRF_SO_SEC_DTLS_CID_ENABLED` - The connection ID extension with a valid CID is included in the client hello. Hence, DTLS connection ID support is requested by the modem.
 
    The default is disabled.
 
 NRF_SO_SEC_DTLS_CID_STATUS
    Get the status of DTLS connection ID.
+   This socket option tells whether the connection ID is used in the current connection.
```

Comment:
At the moment, these lines seem to say the same thing. Is it still correct if Line 290 reads "The status tells whether..."?